app/controllers/api/v2/discovery_rules_controller.rb

```diff
@@ -31,7 +31,7 @@ def show
           param :name, String, :required => true
           param :search, String, :required => true
           param :hostgroup_id, Integer, :required => true
-          param :hostname, String, :required => true
+          param :hostname, String
           param :max_count, Integer
```

Comment:
@tstrachota I didn't want to change the model which already uses `:max_count` but the UI shows `Hosts Limit` so I wanted the api to make sense